tool/fix-urls-for-static-hosting.ts

```diff
@@ -0,0 +1,18 @@
+import child_process, { spawnSync } from "node:child_process";
+import fs from "node:fs";
+import util from "node:util";
+
+import { BUILD_OUT_ROOT } from "../libs/env/index.js";
+
+export function runFixUrlsForStaticHosting(options) {
```

Comment:
```suggestion
export function runFixUrlsForStaticHosting(options) {
```
Should we remove the `options` argument from the signature? [And from the call site(s)…]